wordvecspace/server.py

```diff
@@ -123,20 +123,20 @@ def get_occurrences(self, words_or_indices: list) -> list:
         return res
 
     def get_vector(
-        self, word_or_index: Union[str, int], normalized: bool = False
+        self, word_or_index: Union[str, int], norm: bool = False
```

Comment:
@arrangeesh @RamanjaneyuluIdavalapati we should have backward compatibility this will break the previous code
what is the necessity of this change?

Reply:
@jaswanth098 In few cases it is `normalized` and in few cases it is `normalised`. And even it is confusing whether the input vector is normalised or hosted vectors are normalised. So as we thought it is not a proper name, we used `norm` (Of course we can pick a better name but in a given time constraint we are able pick this name).

And also we thought it won't make problem because most of the users are following versioning. So we renamed it.

Reply:
@RamanjaneyuluIdavalapati it doesn't matter if users use versioning or not, it is primary principal that we should maintain backward compatibility, please have the name as is.

Reply:
@jaswanth098 I hope it is not a strict rule to maintain backward compatibility. If that is the case we don't have any chance to correct the flag name.

Reply:
@RamanjaneyuluIdavalapati I agree that we can't correct things following such rule but normalization is being used in the functions which are exposed to the outside world,
`normalisation` is used only in one place that too is an underscore function which most people might not be using then, in this case, we should change `normalisation` to `normalization` but not change whole variable right
It is better to follow some rules instead break them for such small cases

Reply:
Makes sense to convert all `normalised` -> `normalized` as fix. Both renaming to `normalized` and to `norm` will break backward compatibility but the former is smaller damage.